sample-app/build.gradle.kts

```diff
@@ -33,6 +33,11 @@ android {
     buildFeatures {
         viewBinding = true
     }
+    packagingOptions {
+        jniLibs {
+            useLegacyPackaging = true
+        }
+    }
```

Comment:
This is for building apps with a 16KB page size in the current AGP version.
- https://developer.android.com/guide/practices/page-sizes#agp_version_85_or_lower